test/components/gridlinesTests.ts

```diff
@@ -1,62 +1,62 @@
 ///<reference path="../testReference.ts" />
 
 describe("Gridlines", () => {
-
-  it("Scale ranges are set to the Gridlines dimensions when layout is computed", () => {
-    let svg = TestMethods.generateSVG(640, 480);
-    let xScale = new Plottable.Scales.Linear();
+  const SVG_WIDTH = 640;
+  const SVG_HEIGHT = 480;
+  let svg: d3.Selection<void>;
+  let xScale: Plottable.Scales.Linear;
+  let yScale: Plottable.Scales.Linear;
+  let gridlines: Plottable.Components.Gridlines;
+
+  beforeEach(() => {
+    svg = TestMethods.generateSVG(640, 480);
```

Comment:
Should probably use the `const`s here, but I don't think this test is even dependent on the dimensions, so we can probably go with the defaults.